editor/src/components/canvas/dom-walker.ts

```diff
@@ -1365,3 +1365,19 @@ function walkElements(
     return { childPaths: [], rootMetadata: {}, cachedPaths: [] }
   }
 }
+
+function getClosestOffsetParent(element: HTMLElement): Element | null {
+  let currentElement: HTMLElement | null = element
+
+  while (currentElement != null) {
+    const currentElementStyle = window.getComputedStyle(currentElement)
+    if (isElementAContainingBlockForAbsolute(currentElementStyle)) {
```

Comment:
IIUC this means `getClosestOffsetParent` may sometimes return the `element` and not any parent. that means the function's name is a lie!